.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea/
+.*.swp
+.*.swo
+
+LOCK
+db/
+last_tikv.toml
+raft/
+tidb-slow.log
```

Comment:
Could we move this into another pr and copy the `tikv/.gitignore` into this file?